### DISCUSSION TITLE
Common statusline for different windows?

### DISCUSSION THREAD

**Discussion by ubaldot** [vim/vim#15702] [Ideas]
Not sure if that has been discussed in the past, but what about having a single statusline that span over the whole screen and that change its content based on the active window?

**Comment by chrisbra** [comment]
it's called the tabline

**Comment by ubaldot** [comment]
ah right! 
I was wondering to a sort of 'laststatus = 3' setting.     
This because I have the tabline already occupied with a fairly common "bufline" setting that "show all the listed buffers and highlight the active one" and having a common statusline would be handy.

**Comment by ubaldot** [comment]
Update: Funnily enough, I just noticed that this very request is present in neovim. Can't we just pick the PR that implements it and merge into Vim? Or perhaps there is some rationale why we don't have it in Vim? 

Just wondering :)

**Comment by lifepillar** [comment]
Previous discussion: https://github.com/vim/vim/issues/10153.

**Reply by ubaldot** [reply]
Ah! There we go! Thanks!
